# German translation for linuxmint
# Copyright (c) 2009 Rosetta Contributors and Canonical Ltd 2009
# This file is distributed under the same license as the linuxmint package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2009.
#
msgid ""
msgstr ""
"Project-Id-Version: linuxmint\n"
"Report-Msgid-Bugs-To: FULL NAME <EMAIL@ADDRESS>\n"
"POT-Creation-Date: 2016-11-02 13:58+0000\n"
"PO-Revision-Date: 2016-11-02 22:30+0000\n"
"Last-Translator: Tobias Bannert <tobannert@gmail.com>\n"
"Language-Team: German <de@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Launchpad-Export-Date: 2017-06-28 07:44+0000\n"
"X-Generator: Launchpad (build 18416)\n"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:111
#: usr/lib/linuxmint/mintbackup/mintbackup.py:251
#: usr/lib/linuxmint/mintbackup/mintbackup.py:443
#: usr/lib/linuxmint/mintbackup/mintbackup.py:461
#: usr/lib/linuxmint/mintbackup/mintbackup.py:527
#: usr/lib/linuxmint/mintbackup/mintbackup.py:530
#: usr/lib/linuxmint/mintbackup/mintbackup.py:582
#: usr/lib/linuxmint/mintbackup/mintbackup.py:585
#: usr/lib/linuxmint/mintbackup/mintbackup.py:605
#: usr/lib/linuxmint/mintbackup/mintbackup.py:628
#: usr/lib/linuxmint/mintbackup/mintbackup.py:635
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1499
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1597
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1605
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1661 generate_desktop_files.py:26
#: generate_desktop_files.py:41
msgid "Backup Tool"
msgstr "Datensicherungswerkzeug"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:168
msgid "Preserve structure"
msgstr "Struktur erhalten"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:170
msgid ".tar file"
msgstr ".tar-Datei"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:171
msgid ".tar.bz2 file"
msgstr ".tar.bz2-Datei"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:172
msgid ".tar.gz file"
msgstr ".tar.gz-Datei"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:178
msgid "Never"
msgstr "Niemals"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:179
msgid "Size mismatch"
msgstr "Größe stimmt nicht überein"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:180
msgid "Modification time mismatch"
msgstr "Änderungsdatum stimmt nicht überein"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:181
msgid "Checksum mismatch"
msgstr "Prüfsummenfehler"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:182
msgid "Always"
msgstr "Immer"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:204
msgid "Excluded paths"
msgstr "Ausgeschlossene Pfade"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:214
msgid "Type"
msgstr "Typ"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:218
msgid "Detail"
msgstr "Detail"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:224
#: usr/lib/linuxmint/mintbackup/mintbackup.py:233
msgid "Path"
msgstr "Pfad"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:227
#: usr/lib/linuxmint/mintbackup/mintbackup.py:236
msgid "Error"
msgstr "Fehler"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:268
msgid "Store?"
msgstr "Sichern?"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:271
#: usr/lib/linuxmint/mintbackup/mintbackup.py:284
msgid "Name"
msgstr "Bezeichnung"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:281
msgid "Install"
msgstr "Installieren"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:289
msgid "Backup or Restore"
msgstr "Sichern oder Wiederherstellen"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:290
msgid "Choose from the following options"
msgstr "Bitte aus den folgenden Optionen auswählen"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:292
#: usr/lib/linuxmint/mintbackup/mintbackup.py:303
#: usr/lib/linuxmint/mintbackup/mintbackup.py:317
#: usr/lib/linuxmint/mintbackup/mintbackup.py:324
#: usr/lib/linuxmint/mintbackup/mintbackup.py:328
#: usr/lib/linuxmint/mintbackup/mintbackup.py:333
msgid "Backup files"
msgstr "Dateien sichern"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:293
#: usr/lib/linuxmint/mintbackup/mintbackup.py:337
#: usr/lib/linuxmint/mintbackup/mintbackup.py:347
#: usr/lib/linuxmint/mintbackup/mintbackup.py:353
#: usr/lib/linuxmint/mintbackup/mintbackup.py:358
msgid "Restore files"
msgstr "Dateien wiederherstellen"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:294
#: usr/lib/linuxmint/mintbackup/mintbackup.py:362
#: usr/lib/linuxmint/mintbackup/mintbackup.py:367
#: usr/lib/linuxmint/mintbackup/mintbackup.py:373
#: usr/lib/linuxmint/mintbackup/mintbackup.py:378
msgid "Backup software selection"
msgstr "Anwendungsauswahl sichern"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:295
#: usr/lib/linuxmint/mintbackup/mintbackup.py:382
#: usr/lib/linuxmint/mintbackup/mintbackup.py:387
#: usr/lib/linuxmint/mintbackup/mintbackup.py:394
msgid "Restore software selection"
msgstr "Anwendungsauswahl wiederherstellen"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:297
msgid "Make a backup of your files"
msgstr "Eine Sicherung Ihrer Dateien erstellen"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:298
msgid "Save the list of installed applications"
msgstr "Liste der installierten Anwendungen speichern"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:299
msgid "Restore a previous backup"
msgstr "Eine frühere Sicherung wiederherstellen"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:300
msgid "Restore previously installed applications"
msgstr "Zuvor installierte Anwendungen wiederherstellen"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:304
msgid "Please select a source and a destination for your backup"
msgstr "Bitte eine Quelle und ein Ziel für Ihre Sicherung auswählen"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:305
#: usr/lib/linuxmint/mintbackup/mintbackup.py:341
#: usr/lib/linuxmint/mintbackup/mintbackup.py:349
msgid "Source:"
msgstr "Quelle:"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:306
#: usr/lib/linuxmint/mintbackup/mintbackup.py:342
msgid "Destination:"
msgstr "Ziel:"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:307
#: usr/lib/linuxmint/mintbackup/mintbackup.py:343
msgid "Advanced options"
msgstr "Erweiterte Einstellungen"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:308
#: usr/lib/linuxmint/mintbackup/mintbackup.py:350
msgid "Description:"
msgstr "Beschreibung:"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:309
msgid "Output:"
msgstr "Ausgabe:"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:310
#: usr/lib/linuxmint/mintbackup/mintbackup.py:344
msgid "Overwrite:"
msgstr "Überschreiben:"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:311
msgid "Confirm integrity"
msgstr "Unversehrtheit bestätigen"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:312
msgid "Follow symlinks"
msgstr "Symbolischen Verknüpfungen folgen"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:313
msgid "Preserve permissions"
msgstr "Berechtigungen beibehalten"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:314
msgid "Preserve timestamps"
msgstr "Zeitstempel beibehalten"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:318
msgid "Please select any files or directories you want to exclude"
msgstr ""
"Bitte die Dateien und Ordner auswählen, welche Sie ausschließen möchten"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:319
msgid "Exclude files"
msgstr "Dateien ausschließen"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:320
msgid "Exclude directories"
msgstr "Ordner ausschließen"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:321
msgid "Remove"
msgstr "Entfernen"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:325
msgid "Please review the information below before starting the backup"
msgstr ""
"Bitte die folgenden Informationen überprüfen, bevor Sie die Sicherung "
"ausführen"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:329
msgid "Please wait while your files are being backed up"
msgstr "Bitte warten, während Ihre Dateien gesichert werden"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:330
#: usr/lib/linuxmint/mintbackup/mintbackup.py:375
msgid "Backing up:"
msgstr "Sicherung läuft:"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:334
#: usr/lib/linuxmint/mintbackup/mintbackup.py:379
msgid "The backup is now finished"
msgstr "Die Sicherung ist jetzt abgeschlossen"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:338
msgid ""
"Please choose the type of backup to restore, its location and a destination"
msgstr ""
"Bitte die Art, den Ursprungsort und das Ziel der wiederherzustellenden "
"Datensicherung auswählen."

#: usr/lib/linuxmint/mintbackup/mintbackup.py:339
msgid "Archive"
msgstr "Archiv"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:340
msgid "Directory"
msgstr "Verzeichnis"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:348
msgid "Please review the information below"
msgstr "Bitte die folgenden Informationen überprüfen"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:354
msgid "Please wait while your files are being restored"
msgstr "Bitte warten, während Ihre Dateien wiederhergestellt werden"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:355
msgid "Restoring:"
msgstr "Wiederherstellung läuft:"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:359
msgid "The restoration of the files is now finished"
msgstr "Die Wiederherstellung der Dateien ist nun abgeschlossen"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:363
msgid "Please choose a destination"
msgstr "Bitte ein Ziel auswählen"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:364
#: usr/lib/linuxmint/mintbackup/mintbackup.py:546
msgid "Destination"
msgstr "Ziel"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:368
msgid "The list below shows the packages you added to Linux Mint"
msgstr ""
"Die folgende Liste zeigt die Pakete, die Sie zu Linux Mint hinzugefügt haben"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:369
#: usr/lib/linuxmint/mintbackup/mintbackup.py:389
msgid "Select all"
msgstr "Alles auswählen"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:370
#: usr/lib/linuxmint/mintbackup/mintbackup.py:390
msgid "Deselect all"
msgstr "Die gesamte Auswahl aufheben"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:374
msgid "Please wait while your software selection is being backed up"
msgstr "Bitte warten, während Ihre Anwendungsauswahl gesichert wird"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:383
msgid "Please select a saved software selection"
msgstr "Bitte eine gesicherte Anwendungsauswahl auswählen"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:384
msgid "Software selection:"
msgstr "Anwendungsauswahl:"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:388
msgid "Select the packages you want to install"
msgstr "Bitte die Pakete auswählen, die Sie installieren möchten"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:391
msgid "Refresh"
msgstr "Aktualisieren"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:395
msgid "The restoration is now finished"
msgstr "Die Wiederherstellung ist nun abgeschlossen"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:396
msgid "Your package selection was restored successfully"
msgstr "Ihre Paketauswahl wurde erfolgreich wiederhergestellt"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:452
#: usr/lib/linuxmint/mintbackup/mintbackup.py:470
msgid "Invalid path"
msgstr "Ungültiger Pfad"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:452
#: usr/lib/linuxmint/mintbackup/mintbackup.py:470
#, python-format
msgid "%s is not located within your source directory. Not added."
msgstr "%s befindet sich nicht in Ihrem Quellverzeichnis. Nicht hinzugefügt."

#: usr/lib/linuxmint/mintbackup/mintbackup.py:527
msgid "Please choose directories for the source and the destination"
msgstr "Bitte Ordner für die Quelle und das Ziel auswählen"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:530
msgid ""
"Please choose different directories for the source and the destination"
msgstr ""
"Bitte unterschiedliche Verzeichnisse, für die Quelle und das Ziel, auswählen"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:545
msgid "Source"
msgstr "Quelle"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:548
msgid "Description"
msgstr "Beschreibung"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:552
msgid "Compression"
msgstr "Komprimierung"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:556
msgid "Overwrite destination files"
msgstr "Zieldateien überschreiben"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:559
msgid "Exclude"
msgstr "Ausschließen"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:582
msgid "Please choose a file to restore from"
msgstr "Bitte eine Datei zur Wiederherstellung auswählen"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:585
#: usr/lib/linuxmint/mintbackup/mintbackup.py:605
msgid "Please choose a destination directory"
msgstr "Bitte ein Zielverzeichnis auswählen"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:628
msgid "No packages need to be installed at this time"
msgstr "Zur Zeit müssen keine Pakete installiert werden"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:635
msgid "Please select one or more packages to install"
msgstr "Bitte ein oder mehrere Pakete zum installieren auswählen"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:703
#: usr/lib/linuxmint/mintbackup/mintbackup.py:704
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1206
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1208
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1514
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1515
msgid "Calculating..."
msgstr "Berechnung läuft …"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:719
#, python-format
msgid "Skipping %(skipped_file)s because named pipes are not supported."
msgstr ""
"%(skipped_file)s wird übersprungen, da benannte Leitungen nicht unterstützt "
"werden."

#: usr/lib/linuxmint/mintbackup/mintbackup.py:761
#: usr/lib/linuxmint/mintbackup/mintbackup.py:801
msgid "Skipping broken link"
msgstr "Kaputte Verknüpfung wird übersprungen"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:762
msgid "Broken link"
msgstr "Kaputte Verknüpfung"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:765
#: usr/lib/linuxmint/mintbackup/mintbackup.py:805
msgid "Skipping link"
msgstr "Verknüpfung wird übersprungen"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:829
#: usr/lib/linuxmint/mintbackup/mintbackup.py:838
#: usr/lib/linuxmint/mintbackup/mintbackup.py:847
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1267
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1278
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1291
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1355
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1364
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1374
msgid "Skipping identical file"
msgstr "Identische Datei wird übersprungen"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:869
#, python-format
msgid ""
"Warning: Some files were not saved, copied: %(current_file)d files out of "
"%(total)d total"
msgstr ""
"Achtung: Einige Dateien wurden nicht gespeichert. %(current_file)d Dateien "
"von insgesamt %(total)d wurden kopiert"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:873
msgid "An error occurred during the backup"
msgstr "Ein Fehler ist während der Sicherung aufgetreten"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:883
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1560
msgid "The backup was aborted"
msgstr "Die Sicherung wurde abgebrochen"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:891
msgid "The backup completed successfully"
msgstr "Die Sicherung wurde erfolgreich abgeschlossen"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:982
msgid "Checksum Mismatch:"
msgstr "Prüfsummenfehler:"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:983
msgid "Checksum Mismatch"
msgstr "Prüfsummenfehler"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1035
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1037
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1066
msgid "Calculating checksum"
msgstr "Prüfsumme wird berechnet"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1112
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1136
msgid "(Not created with the Backup Tool)"
msgstr "(Nicht mit dem Wiederherstellungswerkzeug erstellt)"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1398
#, python-format
msgid ""
"Warning: Some files were not restored, copied: %(current_file)d files out of "
"%(total)d total"
msgstr ""
"Achtung: Einige Dateien wurden nicht wiederhergestellt. %(current_file)d "
"Dateien von %(total)d insgesamt kopiert"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1401
msgid "An error occurred during the restoration"
msgstr "Ein Fehler ist während der Wiederherstellung aufgetreten"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1412
msgid "The restoration was aborted"
msgstr "Die Wiederherstellung wurde abgebrochen"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1420
msgid "The restoration completed successfully"
msgstr "Die Wiederherstellung wurde erfolgreich abgeschlossen"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1551
msgid "An error occurred during the backup:"
msgstr "Während der Sicherung trat folgender Fehler auf:"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1568
msgid "Your software selection was backed up successfully"
msgstr "Ihre Programmauswahl wurde erfolgreich gesichert"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1597
msgid "The specified file is not a valid software selection"
msgstr "Die ausgewählte Datei ist keine gültige Anwendungsauswahl"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1605
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1661
msgid "An error occurred while accessing the file"
msgstr "Während des Dateizugriffs trat ein Fehler auf"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1648
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1654
msgid "Could not locate the package"
msgstr "Das Paket konnte nicht gefunden werden"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1686
msgid "Please wait, this can take some time"
msgstr ""
"Bitte gedulden Sie sich. Dieser Vorgang kann einige Zeit in Anspruch nehmen."

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1688
msgid "The installation is complete"
msgstr "Installation abgeschlossen"

#: generate_desktop_files.py:26 generate_desktop_files.py:41
msgid "Make a backup of your home directory"
msgstr "Eine Sicherung Ihres persönlichen Ordners erstellen"

#~ msgid "Backup"
#~ msgstr "Datensicherung"

#~ msgid "Restoration failed"
#~ msgstr "Wiederherstellung fehlgeschlagen"

#~ msgid "Read error"
#~ msgstr "Lesefehler"

#~ msgid "Hidden paths"
#~ msgstr "Versteckte Pfade"

#~ msgid "<big><b>Load data into your home directory</b></big>"
#~ msgstr "<big><b>Lade Daten in Ihren persönlichen Ordner</b></big>"

#~ msgid "is not located within your home directory. Not added."
#~ msgstr "ist nicht in Ihrem persönlichen Ordner enthalten. Nicht hinzugefügt."

#~ msgid "Included hidden directories"
#~ msgstr "Enthaltene versteckte Verzeichnisse"

#~ msgid "Included"
#~ msgstr "Enthalten"

#~ msgid "Restore"
#~ msgstr "Wiederherstellen"

#~ msgid "View content"
#~ msgstr "Inhalt ansehen"

#~ msgid "Your home directory was successfully backed-up into"
#~ msgstr "Ihr persönliches Verzeichnis wurde erfolgreich gesichert nach"

#~ msgid "Backup successful"
#~ msgstr "Sicherung erfolgreich"

#~ msgid "Backup failed"
#~ msgstr "Sicherung fehlgeschlagen"

#~ msgid "Extracting the content of the backup into your home directory..."
#~ msgstr "Entpacke den Inhalt der Sicherung in Ihr persönliches Verzeichnis..."

#~ msgid "Copying the backup file into your home directory..."
#~ msgstr "Kopiere die Sicherungsdatei in Ihr persönliches Verzeichnis..."

#~ msgid "Restoration successful"
#~ msgstr "Wiederherstellung erfolgreich"

#~ msgid "An error occurred while restoring the backup archive:"
#~ msgstr ""
#~ "Bei der Wiederherstellung des Sicherungsarchivs trat folgender Fehler auf:"

#~ msgid "An error occurred while opening the backup:"
#~ msgstr "Beim Öffnen der Sicherung trat folgender Fehler auf:"

#~ msgid "Your backup was successfully restored"
#~ msgstr "Ihre Daten wurden erfolgreich wiederhergestellt"

#~ msgid "Restore your personal data from this backup"
#~ msgstr ""
#~ "Stellen Sie Ihre persönlichen Daten aus dieser Sicherungskopie wieder her"

#~ msgid "Excluded Files and Directories"
#~ msgstr "Ausgeschlossene Dateien und Verzeichnisse"

#~ msgid "Exclude folders"
#~ msgstr "Ordner ausschließen"

#~ msgid "Archiving your home directory..."
#~ msgstr "Archiviere den persönlichen Ordner..."

#~ msgid "Cleaning up..."
#~ msgstr "Säuberung läuft …"

#~ msgid "Opening the backup archive..."
#~ msgstr "Sicherungsarchiv wird geöffnet …"

#~ msgid "Overwrite existing files"
#~ msgstr "Bestehende Dateien überschreiben"
